inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cluster/InlongClusterServiceImpl.java

```diff
@@ -218,6 +229,9 @@ public Boolean deleteTag(Integer id, String operator) {
             LOGGER.error("inlong cluster tag not found by id={}", id);
             return false;
         }
+        String[] inChargesArr = exist.getInCharges().split(",");
+        Preconditions.checkTrue(isInCharge(inChargesArr, operator),
```

Comment:
The `String[] inChargesArr = exist.getInCharges().split(",");` can be merged into `isInCharge()` method.

BTW, the `isInCharge()` method seems can refactor as a util method.

Reply:
The `exist` have  many types, like `InlongClusterEntity`、`InlongClusterNodeEntity`、`InlongClusterTagEntity`.